tests/unit/pureHelpers.spec.ts

```diff
@@ -126,7 +83,7 @@ describe("can convert TKN amount to wei with correct precision and rounding", ()
   });
 });
 
-describe("calculate grouped positions for protected table", () => {
+/*describe("calculate grouped positions for protected table", () => {
```

Comment:
Is this test no longer required?

Reply:
The whole fee calculation formula has been retired. New formula: Fully Protected - Inital Stake = Fee  ... and that is it.